generator/uscore/uscore_generator.rb

```diff
@@ -377,6 +377,38 @@ def create_resource_profile_test(sequence)
         sequence[:tests] << test
       end
 
+      def create_multiple_or_test(sequence)
+        test = {
+          tests_that: 'The server returns expected results when parameters use composite-or',
+          index: sequence[:tests].length + 1,
+          link: sequence[:profile],
+          test_code: ''
+        }
+
+        multiple_or_params = sequence[:search_param_descriptions]
```

Comment:
This seems like it could be its own method.

Reply:
Moved to its own method

Reply:
You should use that method here 😉

Reply:
fixed